src/main/scala/io/github/spark_redshift_community/spark/redshift/RedshiftCatalog.scala

```diff
@@ -45,24 +46,17 @@ class RedshiftCatalog extends JDBCTableCatalog {
         }
       }
   
-  private def getTableName(ident: Identifier): String = {
-    (ident.namespace() :+ ident.name()).map(dialect.quoteIdentifier).mkString(".")
-  }
   override def invalidateTable(ident: Identifier): Unit = {
-    // TODO  When refresh table, then drop the s3 folder
+    import org.apache.hadoop.fs.{FileSystem, Path}
+    import java.net.URI
```

Comment:
curious why importing here instead of on the module like usual.

Reply:
Well in case I d'like to move the logic somewhere else this will help a bit. Nothing really important